docs/line.asciidoc

```diff
@@ -27,9 +27,27 @@ NOTE: In Elasticsearch releases 1.4.3 and later, this functionality requires you
 {ref}/modules-scripting.html[dynamic Groovy scripting].
 
 The availability of these options varies depending on the aggregation you choose.
-Select *view options* to change the following aspects of the table:
 
+Select the *Options* tab to change the following aspects of the chart:
+
+*Y-Axis Scale*:: You can select *linear*, *log*, or *square root* scales for the chart's Y axis.
+*Smooth Lines*:: Check this box to curve the line from point to point.
+*Show Connecting Lines*:: Check this box to draw lines between the points on the chart.
+*Show Circles*:: Check this box to draw each data point on the chart as a small circle.
 *Show Tooltip*:: Check this box to enable the display of tooltips.
 *Show Legend*:: Check this box to enable the display of a legend next to the chart.
-*Scale Y-Axis to Data Bounds*:: The default Y axis bounds are zero and the maximum value returned in the data. Check 
-this box to change both upper and lower bounds to match the values returned in the data.
\ No newline at end of file
+*Scale Y-Axis to Data Bounds*:: The default Y-axis bounds are zero and the maximum value returned in the data. Check 
+this box to change both upper and lower bounds to match the values returned in the data.
+
```

Comment:
Check out the y-min/y-max pull, you'll need to add stuff around it to this list: https://github.com/elastic/kibana/pull/3464